test/support/style_case.ex

```diff
@@ -46,7 +49,10 @@ defmodule Styler.StyleCase do
       end
 
       assert expected == styled
-      {_, restyled, _} = style(styled)
+
+      if @strict do
+        assert styled_ast == expected_ast
+      end
```

Comment:
i think i could've accomplished the same by just adding the comments in sooner... so probably nix this before merging